src/index.ts

```diff
@@ -11,11 +11,13 @@ import { getApiToken } from "./helpers/configHelpers";
 	await getOrLoadAllCommands(false);
 
 	const CLIENT = new Client({ intents: [Intents.FLAGS.GUILDS, Intents.FLAGS.GUILD_MESSAGES] });
-	CLIENT.once("ready", () => console.log("Ready!"));
+	CLIENT.once("ready", async () => {
+		CLIENT.user?.setActivity("/help", { type: "WATCHING" });
```

Comment:
This would be better to [specify when creating `CLIENT`](https://discord.js.org/#/docs/discord.js/stable/typedef/ClientOptions) - IIRC there's currently a bug in Discord's API where activities are cleared after a bit, but it doesn't happen if specified when logging into the gateway.

Reply:
Thank you! I moved it to the client constructor